remoteappmanager/db/csv_db.py

```diff
@@ -30,7 +30,7 @@
     If undefined, common data volume is not available
 
 policy.volume_mode : str
-   Mode for read/write access (ro: read-only, rw: read-write)
```

Comment:
This wasn't needed, I already fixed it in the other branch. We'll get a conflict but we'll fix it. Leave it as is.